packages/swingset-liveslots/src/collectionManager.js

```diff
@@ -48,6 +48,19 @@ function throwNotDurable(value, slotIndex, serializedValue) {
   );
 }
 
+/**
+ *
+ * @param {*} syscall
+ * @param {import('./virtualReferences.js').VirtualReferenceManager} vrm
+ * @param {() => number} allocateExportID
+ * @param {() => number} allocateCollectionID
+ * @param {(val: any) => string | undefined} convertValToSlot
+ * @param {*} convertSlotToVal
```

Comment:
when is `{*}` better than `unknown`?

Reply:
`{*}` is `{any}`. `unknown` is better but some declaration is better than nothing